### ISSUE TITLE
[Frontend_v2] Input field colors changes to white after autofill

### ISSUE THREAD

**Issue by ShauryaAg** [Cloud-CV/EvalAI#3245]
Input Field color changes to white after autofill in `template-challenge-create`

If you'd like to report a bug, please include the following information:
#### The expected behaviour
color should remain `black`
#### The observed behaviour
color changes to `white` after autofill

#### Steps to reproduce the problem
1. Create a new challenge using a template
2. Use autocomplete to fill `Challenge Title` or `Challenge Description` field
3. Observe

Refer to the GIF below for additional context:
![localhost_4200_template-challenge-create_1_1_gif](https://user-images.githubusercontent.com/31778302/107873338-c8c04580-6ed7-11eb-960e-8066496eae7d.gif)

**I would like to work on this issue.**

**Comment by ShauryaAg** [comment]
@Ram81 Is this behavior by choice and expected? 

Seems like it is from this code spinnet:

https://github.com/Cloud-CV/EvalAI/blob/5a5c28abb7687a82e18379e0f02a02d3748d1bf8/frontend_v2/src/styles/base.scss#L1625-L1633

**Comment by Ram81** [comment]
@ShauryaAg no, this is not the expected behavior. If you are interested you could pick this issue. Thanks for reporting it 🙂

**Comment by ShauryaAg** [comment]
@Ram81 Sure, I would like to work on this issue.

**Comment by Ram81** [comment]
@ShauryaAg sure, go ahead. Also, it would be good if you look at the comments on this [PR](https://github.com/Cloud-CV/EvalAI/pull/3234)
